src/core_plugins/kibana/public/dashboard/grid/dashboard_grid_container.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
+import { DashboardGrid } from './dashboard_grid';
+import { updatePanel } from '../actions';
+
+const mapStateToProps = ({ dashboardState }, ownProps) => {
+  return {
+    panels: dashboardState.panels,
+    getContainerApi: ownProps.getContainerApi,
```

Comment:
These `ownProps` forwards will automatically be injected, so there's no need to specify them. The result of `mapStateToProps` is merged with `props`.

Reply:
swwweeeeet